addon/templates/components/footer.hbs

```diff
@@ -0,0 +1,70 @@
+<footer class="c-main-footer" data-test-main-footer ...attributes>
+
+  <div class="l-container u-spacing--triple">
+    <div class="l-grid l-grid--2up l-grid--large-gutters">
+      <div>
+        <div class="c-main-footer__logo u-space--double--bottom">
```

Comment:
This is more of something we want to fix in the long term but i'd like to refactor or remove these utility classes.
Like what is 'space double'? is that our --space-2 token? something else? I think it's tied to our legacy tokens but i don't know what it corresponds to now. In any case, I'd prefer have these divs with classes that describe their purpose and add the appropriate tokens/styles in the css. But i think this is something to get to later, just pointing it out.

Reply:
These spacing classes have the new space tokens... double is (--space-3) * 2. They were already there so I've been using them. I think having spacing utility classes is a good thing. In the example above on line 3 we would have needed to write a special class for the container otherwise.

Reply:
I wrote a long comment here but osx update restarted my computer without warning and wiped it out before i could submit, might try again tomorrow